packages/react/src/components/Tag/Tag.js

```diff
@@ -53,8 +58,10 @@ const Tag = ({
       <span className={`${prefix}--tag__label`}>
         {children !== null && children !== undefined ? children : TYPES[type]}
       </span>
-      <Close16 />
-    </button>
+      <button className={`${prefix}--tag__close-icon`} onClick={handleClose}>
```

Comment:
With the button shifting over will we also need to move the `aria-label` over?

Reply:
not sure but the aria label was removed from the close button to the tag body in #4863